sys/vane/jael.hoon

```diff
@@ -1064,11 +1315,36 @@
     vein:feel
   ::                                                    ::  ++meet:su
   ++  meet                                              ::  seen after breach
-    |=  who=ship
+    |=  [who=ship =life =pass]
     ^+  +>
-    =+  ~|  [%met-unknown-ship who]
-        (~(got by kyz.puk) who)
-    (pubs:feel [[who -(live &)] ~ ~])
+    =;  new=public
+      (pubs:feel (my [who new] ~))
+    ::
+    =/  old=(unit public)
+      (~(get by kyz.puk) who)
+    ?:  ?|  ?=(?(%earl %pawn) (clan:title who))
+            ::  XX save %dawn sponsor in .own.sub, check there
+            ::  XX or move sein:of to sein:su?
+            ::  XX full saxo chain?
```

Comment:
Why would a full sponsorship chain need to be seen here, or anywhere really?

Reply:
This is about meeting ships when we don't yet have their keys. This is always the case for moons and comets, and otherwise the case for our sponsor. If the "galaxy table" we supply in the boot event included the sponsor's keys and/or full sponsorship chain, this special casing (and related logic in %ames) would not be necessary.